jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -273,6 +274,10 @@ def _selected_data_changed(self, event):
     @observe("spectral_subset_selected")
     def _on_spectral_subset_selected(self, event):
         # If "Entire Spectrum" selected, reset based on bounds of selected data
+        if self._spectrum1d is None:
+            # TODO: this should be removed as soon as the data dropdown component is
```

Comment:
Is there a follow-up issue for this?

Reply:
yes, although with different behavior in each plugin, refactoring to remove all this logic might be quite involved. [:cat:](https://jira.stsci.edu/browse/JDAT-2221)